### PR TITLE
Add `quorumModifier` to `balance` plugin

### PR DESCRIPTION
**Pull request by RatanRSur**
Currently, only `multichainBalance` has a `quorumModifier` option. This is useful for the `balance` case as well, however. This PR pulls the value up one level so it can be used in both places.

### CODE REVIEW

**Review by ChaituVR — APPROVED** [review]
LGTM, It is the same as multichainBalance, But would like to have a second review from @samuveth @mktcode @bonustrack 

Is there a chance that `quorumModifier` was passed into the `static` type in some proposals accidentally? I don't think it is possible but just in some rare case if someone passed it before, this could be a problem

**Comment by RatanRSur** [comment]
Even if they did pass it in, `static` doesn't multiply by `quorumModifier`. Is anything else needed to merge this in?

**Review by samuveth — APPROVED** [review]

**Pull request merged**
